content/docs/user-guide/interactivity/robotics/importing-robot/sdformat-sensors.md

```diff
@@ -0,0 +1,66 @@
+
+It is important to note, that implementation of sensors in [ROS 2 Gem](/docs/user-guide/gems/reference/robotics/ros2/) exposes ROS 2 communication interfaces by design, which makes it comparable to Gazebo sensor plugins. Therefore, you do not have to define a Gazebo sensor plugin if you want to make ROS 2 topics available. On the other hand, both sensor and plugin names must match the _hook's_ definition to be added to O3DE representation of the robot if the plugin definition exists in the robot description file. This way you can override the default behavior of the Robot Importer with your specific implementation of the same sensor.
+
+### SDF sensor tag mappings
+
+Four basic sensors' _hooks_ are predefined in Robot Importer. These can be summarized as follows:
+
+| _Hook_ name        | Supported SDFormat sensors | Supported SDFormat plugins    | O3DE sensor component       |
+| ------------------ | -------------------------- | ----------------------------- | --------------------------- |
+| _CameraSensorHook_ | `camera_sensor`,           | `libgazebo_ros_camera`        | `ROS2CameraSensorComponent` |
+|                    | `depth_camera`             | `libgazebo_ros_depth_camera`  |                             |
+|                    | `multicamera`              | `libgazebo_ros_multicamera`   |                             |
+|                    | `rgbd_camera`              | `libgazebo_ros_openni_kinect` |                             |
+| _GNSSSensorHook_   | `gps`, `navsat`            | `libgazebo_ros_gps_sensor`    | `ROS2GNSSSensor`            |
+| _ImuSensorHook_    | `imu`                      | `libgazebo_ros_imu_sensor`    | `ROS2ImuSensorComponent`    |
+| _LidarSensorHook_  | `lidar`, `ray`             | `libgazebo_ros_ray_sensor`    | `ROS2LidarSensorComponent`  |
+|                    | `gpu_lidar`, `gpu_ray`     | `libgazebo_ros_laser`         | _see information below_     |
+
+GPU implementation of ray/lidar sensors is not included in ROS 2 Gem and requires additional [O3DE RGL Gem](https://github.com/RobotecAI/o3de-rgl-gem). GPU sensors are mapped during the import to run on a CPU if this Gem is not available.
+
+### Extending default mapping
+
+You can extend the default mapping by implementing additional _hooks_ and registering them in the system based on the _SerializeContext_ reflection system. Typically, this consists of three tasks.
+
+First, you need to declare `ROS2::SDFormat::SensorImporterHook` structure that consists of the following:
+* set of SDFormat sensors associated with your import scheme
+* set of plugin names associated with your import scheme
+* set of the supported parameters in the input robot description (used for import verbose only)
+* registered callback function that is invoked by Robot Importer when _hook's_ definition matches the input data
+
```

Comment:
Insert a code example here of the CreateMyHook() method

I propose to use this name instead of YourMethodToCreateHook (it is more tutorial standard).

Reply:
Added.